src/openforms/appointments/contrib/qmatic/tests/test_api_endpoints.py

```diff
@@ -1,30 +1,44 @@
-from django.test import TestCase
+from unittest.mock import patch
+
 from django.urls import reverse
 
 import requests_mock
+from rest_framework.test import APITestCase
 
 from openforms.logging.models import TimelineLogProxy
 from openforms.submissions.tests.factories import SubmissionFactory
 from openforms.submissions.tests.mixins import SubmissionsMixin
+from openforms.utils.tests.cache import clear_caches
+
+from ....constants import AppointmentDetailsStatus
+from ....models import AppointmentsConfig
+from ....tests.factories import AppointmentInfoFactory
+from ..client import QmaticException
+from .factories import QmaticConfigFactory
+from .test_plugin import mock_response
 
-from ...constants import AppointmentDetailsStatus
-from ...contrib.qmatic.client import QmaticException
-from ...contrib.qmatic.tests.factories import QmaticConfigFactory
-from ...contrib.qmatic.tests.test_plugin import mock_response
-from ...models import AppointmentsConfig
-from ...tests.factories import AppointmentInfoFactory
 
+class MockConfigMixin:
```

Comment:
Could this use the mixin in src/openforms/appointments/contrib/qmatic/tests/utils.py?